src/main/kotlin/app/revanced/patches/youtubevanced/ad/general/patch/HideAdsPatch.kt

```diff
@@ -46,7 +46,9 @@ class HideAdsPatch : BytecodePatch(
                     "video_display_full_layout",
                     "hero_promo_image",
                     "statement_banner",
-                    "primetime_promo"
+                    "primetime_promo",
+                    "carousel_footered_layout",
+                    "feature_grid_interstitial"
```

Comment:
What do these block?

Reply:
YouTube sometimes offers premium trial, it blocks that banner.

Reply:
Two strings were added, what about the other?

Reply:
- `primetime_promo` blocks YouTube TV ads


![image](https://user-images.githubusercontent.com/107796137/229276731-b09bdc55-7a26-436c-89b8-0b412e18ed87.png)